jekyll/_posts/2018-06-03-create-a-simple-macro.md

```diff
@@ -0,0 +1,120 @@
+
+---
+title: "Create a simple macro"
+author: Antonis Geralis
+---
+
```

Comment:
Please add a "Guest Post" div, like done here:

https://nim-lang.org/blog/2018/01/22/yes-command-in-nim.html

Look in the underlying .md to see how it's done.